### ISSUE TITLE
More Features

### ISSUE THREAD

**Issue by AMOfficalGithub** [benawad/vsinder#248]
Been using this app for some time and I think more features would be beneficial:

**Filter based on Country**
This is important because I see a lot of users on this app are either from Europe or another country and not from USA. By putting in a filter option to sort profiles based on Country this will help filter out people who don't want to travel and or move to another country.

**Online Status**
Having vague time-stamps isn't enough. Users that are online should be indicated by a green button, where users off should be grey and 'busy' users should be red.

**De-activate dead accounts**
This is another prominent issue I see with Vsinder. A lot of users have created accounts but have either decided not to log in anymore. By having an account to have an 'active' login for atleast 30 days should be significant enough. Otherwise, the account is de-activated and it is removed from the swiping pile until the user logs back into the account again.

**Filter out garbage profiles**
Yet another issue with VSinder with the immaturity of a lot of users. Some put literal one words in their profiles, silly bios that amount to nothing, and over-all content that shouldn't be on an app designed for dating. By placing in a minimum word limit and filtering out garbage code/words, this will over-all help a better and genuine type of profiles available to match up with instead of the "Lulz I don't know why i'm here" or the "I don't code" or the copy-paste scripts from stack over-flow/geeksforgeeks, etc. This may require moderation.

**Restrict/Remove profiles 50+ with age verification or 18+**
Another issue that should be aware of is unfortunately teenagers have created accounts on this app for 'giggles'. Whether they know this or not, this app is design for those of age 18 and older and there are plenty of young to older adults that use this app. They are usually the default 51 years old age profiles or, of course, fake their age. By requiring verification of age this helps filter out those teenagers that sign up on platforms like this. Can help avoid a lot of legal headaches and making sure minors aren't accessing and using dating apps such as this.

**Add a open-kattis/hacker-rank/leet code verification**
Want to make sure the person that your selecting as a possible date actually knows what their doing and not some copy-paste? By adding these technical assessments, it'll help ensure that the person your talking to isn't some super-beginner that knows 0 code or just someone copying existing code somewhere. The level of difficulty can be adjusted. 

**Time-Stamps per message**
While it's possible to see Time-Stamps, it's usually not until the next day you actually see the time. Have it based by default of region when the last message was sent. This will help coordinate times in different zones and also as an indication of availbility. 

**Remove the 'It's a Joke' app reference**
This app has potential but for it to be taken as a joke is seriously holding it back. Thus the reason why for all the garbage profiles, inactive accounts, immature users, etc. Whoever is in charge of this app now should re-brand it to be an actual dating app meant for programmers across the world. It should be on the level of Tinder, Bumble, and other dating apps but without the annoyance of those main-stream gimmicks and often times, non-IT friendly dating sites/app.

**Comment by unDeadHerbs** [comment]
Some of those are active feature requests already:
- Location:
  - #135 
  - #190
- Online/Offline:
  - #241 
- Filter out garbage profiles:
  - #155 
  - #149
- hacker-rank is similar to:
  - #164 
  - #222 

Could you split these comments out into separate issues for easier discussion?
I broadly agree with some minor comments.

Unfortunately, there doesn't seem to be any developers working on this project.
As far as I can tell, it's just me filtering though the issues when I have some time to set aside.
